benefits/core/templates/core/includes/media-list.html

```diff
@@ -3,18 +3,19 @@
         <li class="media">
             <div class="media-line">{% include "core/includes/icon.html" with icon=item.icon %}</div>
             <div class="media-body">
-                <h3 class="media-body--heading">{{ item.heading }}</h3>
-                <div class="media-body--details">
-                    <p>{{ item.details }}</p>
-                    {% if item.bullets|length %}
-                        <div class="media-body--items">
-                            <ul>
-                                {% for l in item.bullets %}<li>{{ l }}</li>{% endfor %}
-                            </ul>
-                        </div>
-                    {% else %}
-                    {% endif %}
-                </div>
+                {% if item.heading %}<h3 class="media-body--heading">{{ item.heading }}</h3>{% endif %}
```

Comment:
No changes required/just a comment: Technically, the CSS does not have any regressions if an `<h3>` does get rendered with an empty `{{ item.heading }}`. Same with having an empty `<div class="media-body--details>`. The CSS / HTML is written in such a way that if it's empty, the DOM does not create/add any extra height or anything:

<img width="1099" alt="image" src="https://user-images.githubusercontent.com/3673236/193875156-74a83ca6-1377-4f19-8772-cc243fb993ea.png">

Reply:
That's fine... but it's still not semantically correct, especially with an empty heading. We should always avoid rendering empty HTML.